src/shared/mount-setup.c

```diff
@@ -189,13 +190,29 @@ static int mount_one(const MountPoint *p, bool relabel) {
         else
                 (void) mkdir_p(p->where, 0755);
 
+        _cleanup_free_ char *extend_options = NULL;
+        const char *o = p->options;
+        if (FLAGS_SET(p->mode, MNT_TMPFS_USRQUOTA)) {
+                r = mount_option_supported("tmpfs", "usrquota", /* value= */ NULL);
```

Comment:
maybe cache the result given it's generic?

Reply:
i thought about this, but pid1 is using this exactly once... doesn't seem worth it to cache it.